### DISCUSSION TITLE
how can i set the new windows position，and set the existing windows position

### DISCUSSION THREAD

**Discussion by jinzhongjia** [fyne-io/fyne#3247] [Q&A] [answered]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same feature, before opening a new one.
- [X] This issue only relates to a single feature. I will open new issues for any other features.

### Is your feature request related to a problem?

I'm trying to write a gadget that is called out by a shortcut key, it should be at the top of the screen, not in the middle

### Is it possible to construct a solution with the existing API?

I didn't find an api that can set the window position

### Describe the solution you'd like to see.

Requires api to set the position of the new window and change the position of the window that has been shown

**Comment by Jacalz** [comment]
Please use GitHub discussions for asking questions. I will transfer this one to be in the right spot.

**Comment by jinzhongjia** [comment]
My problem, I will pay attention next time

**Comment by Jacalz** [comment, marked as answer]
Sorry for the late answer. This can not be done until https://github.com/fyne-io/fyne/issues/1155 is implemented, if it is implemented. See the discussion on the issue for more information.
